lib/src/widgets/chat/ds_carrousel.widget.dart

```diff
@@ -0,0 +1,118 @@
+import 'package:flutter/material.dart';
+
+import 'package:blip_ds/src/models/ds_document_select.model.dart';
+
+import '../../../blip_ds.dart';
+import '../utils/ds_card.widget.dart';
+
+/// A Design System widget used to display multiple cards.
+///
+/// The widget receives a json passed by the [content] parameter and displays the information
+/// in two ways according to the [itemType] parameter contained in the header.
+/// If the [itemType] contains the select parameter, the content is displayed in carousel mode, and if it
+/// contains the container parameter, it is displayed in a vertical line.
+class DSCarrousel extends StatelessWidget {
+  /// Sets the card's alignment on the screen.
+  final DSAlign align;
+
+  /// Widget content containing card parameters
+  final Map<String, dynamic> content;
+
+  /// Selection return callbacks in menus
+  final void Function(String, Map<String, dynamic>)? onSelected;
+  final void Function(Map<String, dynamic>)? onOpenLink;
+
+  const DSCarrousel({
+    Key? key,
+    required this.align,
+    required this.content,
+    this.onSelected,
+    this.onOpenLink,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return _buildCollection();
+  }
+
+  Widget _buildCollection() {
+    var children = <Widget>[];
+    String typeCollection;
+    var items = [];
+
+    var contentSelect = content['content'];
```

Comment:
could be final

Reply:
https://dart.dev/guides/language/effective-dart/usage#do-follow-a-consistent-rule-for-var-and-final-on-local-variables

I'm following this rule

Use var for all local variables, even ones that aren’t reassigned. Never use final for locals. (Using final for fields and top-level variables is still encouraged, of course.)

Reply:
<img width="930" alt="Screen Shot 2022-10-20 at 16 45 22" src="https://user-images.githubusercontent.com/20003773/197043459-35aeca8b-4189-43cb-8c5b-19ef65aae522.png">

Reply:
let's use the pattern of always using final, what do you think? @jwdutra  @leonardogbr  @mpamaro

Reply:
Done